workers/generic-worker/usage_multiuser_other.go

```diff
@@ -6,6 +6,6 @@ func enableTaskFeatures() string {
 	return `
           enableInteractive                 Enables the Interactive feature to be used in the
                                             task payload. [default: true]
-		  enableRunTaskAsCurrentUser		Enables the Run Task As Current User feature to be
-		  									used in the task payload. [default: true]`
+		  enableRunTaskAsCurrentUser        Enables the Run Task As Current User feature to be
```

Comment:
This still has tabs before `enableRunTaskAsCurrentUser`.

Reply:
Ugh 🤦🏻‍♂️